dbms/src/Common/TiFlashMetrics.h

```diff
@@ -582,6 +582,8 @@ static_assert(RAFT_REGION_BIG_WRITE_THRES * 4 < RAFT_REGION_BIG_WRITE_MAX, "Inva
       F(type_ru_exhausted, {"type", "ru_exhausted"}),                                                                               \
       F(type_add_cache_succ, {"type", "add_cache_succ"}),                                                                           \
       F(type_add_cache_stale, {"type", "add_cache_stale"}),                                                                         \
+      F(type_add_cache_count_limit, {"type", "add_cache_count_limit"}),                                                             \
```

Comment:
```suggestion
      F(type_add_cache_reach_count_limit, {"type", "add_cache_count_limit"}),                                                             \
```

maybe `type_add_cache_reach_count_limit` is better